### ISSUE TITLE
LyoDesigner no longer builds thanks to a recent change to the pom.xml file

### ISSUE THREAD

**Issue by jadelkhoury** [eclipse/lyo.designer#198]
Latest build that fails: 
https://ci.eclipse.org/lyo/job/lyo-designer-master/290/display/redirect?page=changes

The reason is simple: someone change the main.xml file, introducing a new "models" profile.
The change also meant that the "eclipse-sign" profile used during the build process has no longer any modules to build. Why? 
The <modules> are now defined within each profile - except for the "eclipse-sign" profile.

**Comment by berezovskyi** [comment]
Great catch! Fixed in https://ci.eclipse.org/lyo/view/1Enabled/job/lyo-designer-master/291/ via `-Pdefault,eclipse-sign` profile setting.

**Comment by jadelkhoury** [comment]
great. I also assumed you reverted the earlier change to a higher version of mvn?

**Comment by berezovskyi** [comment]
No, I kept maven 3.8. I always used the latest version but there was a critical bug in 3.6.1 if I recall and after long debug we pinned it at 3.6.0 and then to 3.6.3 when a fix was released. I have also used 3.8 with Lyo on my dev machine for many months, I believe, without any problems.

–Andrew

On 3 Sep 2021, at 18:02, Jad El-khoury ***@***.***> wrote:

﻿

great. I also assumed you reverted the earlier change to a higher version of mvn?
